packages/block-editor/src/components/preview-options/index.js

```diff
@@ -33,6 +33,7 @@ export default function PreviewOptions( {
 	const toggleProps = {
 		className: 'block-editor-post-preview__button-toggle',
 		disabled: ! isEnabled,
+		__experimentalIsFocusable: ! isEnabled,
```

Comment:
I have added this prop to make the dropdown focusable even when it is disabled and to show the tooltip. This change will also affect the preview option button in the post editor. Do you think this should not be added in this PR?

Reply:
No, that's fine and a good improvement here.

I just realized that we also need to set `disableOpenOnArrowDown` based on the enabled status to avoid the opening drop via the arrow-down key. Pushed changes here 758e448.

## Screencast


https://github.com/WordPress/gutenberg/assets/240569/bf598b9b-147d-4b9c-a55d-ee8402c64083

Reply:
Thank you for finding it, @Mamaduka!